src/Dev/DevelopmentAdmin.php

```diff
@@ -258,4 +269,40 @@ public function errors()
     {
         $this->redirect("Debug_");
     }
+
+    public function providePermissions(): array
+    {
+        return [
+            'ALL_DEV_ADMIN' => [
+                'name' => _t(__CLASS__ . '.ALL_DEV_ADMIN_DESCRIPTION', 'Can view and run all /dev endpoints'),
+                'help' => _t(__CLASS__ . '.ALL_DEV_ADMIN_HELP', 'Can view and run all endpoints at /dev).'),
```

Comment:
If the help text is required it might as well be the same as the description (vs just swapping two words around).

```suggestion
                'name' => _t(__CLASS__ . '.ALL_DEV_ADMIN_DESCRIPTION', 'Can view and run all /dev endpoints'),
                'help' => _t(__CLASS__ . '.ALL_DEV_ADMIN_HELP', 'Can view and run all endpoints at /dev).'),
```